site/content/api/Tenant-API.md

```diff
@@ -362,6 +362,28 @@ The table below provides an overview of the members defined for the *trusted-ca*
   * Implementations MUST reject requests to add or update a tenant with payload that contains a subject DN that is already configured for another tenant and return a status code of *409 Conflict*.
 * Either the *cert* or the *public-key* MUST be set.
 
+### Examples
+
+Below is an example for a payload of the response to a *get* request for tenant `TEST_TENANT` configured with two trusted certificate authorities.
+
+~~~json
+ {
+   "tenant-id" : "TEST_TENANT",
+   "enabled" : true,
+   "trust-store": [ {
+	    "subject-dn": "CN=ca,OU=Hono,O=Eclipse",
+	    "public-key": "NOTAPUBLICKEY",
+	    "not-before": "2015-01-01T00:00:00+0000",
```

Comment:
neither *not-before* nor *not-after* are part of the content model, are they? And a trust store usually does not contain such properties as well.